.github/workflows/tests.yml

```diff
@@ -23,36 +23,36 @@ jobs:
       fail-fast: true
       matrix:
         os:
-          - "ubuntu-20.04"
+          - "ubuntu-22.04"
```

Comment:
WDYT of running MongoDB 8.0 tests on Ubuntu 24.04? That would give us an extra two years assuming GitHub removes support at roughly the same time frame as they did in this instance.

Reply:
I think that's what I did, but did not write explicitly.

Reply:
Fixed for one missing.